gauss_plot_2d.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #49.        latexify(width_scale_factor=1.5, fig_height=3)
</pre><p>set <code>fig_height=1.5</code> I just checked latex source code. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
I see, I remembered you told me, okay go ahead with 3.

Reply:
But don't change the name of figure when you save, because that name is already referred in latex source code

Reply:
The names have changed because previously 
 Gs = ["Full", "Diag", "Spherical"]
 I have changed it to
Gaussians = ["Full", "Diagonal", "Spherical"] for better understanding, apart from that there are no changes